src/components/TopBar/ProjectPickerButton.jsx

```diff
@@ -1,19 +1,30 @@
-import {faCaretDown} from '@fortawesome/free-solid-svg-icons';
-import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 import classnames from 'classnames';
 import i18next from 'i18next';
+import isEmpty from 'lodash-es/isEmpty';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-export default function ProjectPickerButton({shouldShowSavedIndicator}) {
+export default function ProjectPickerButton({
+  currentProjectKey,
+  isUserAuthenticated,
+  projectKeys,
+  shouldShowSavedIndicator,
+  onClick,
+}) {
+  if (
+    !(
+      isUserAuthenticated &&
+      !isEmpty(projectKeys) &&
+      Boolean(currentProjectKey)
```

Comment:
Similar to previous comment, I think it’s more precise to write `!isNil(currentProjectKey)`

Reply:
Also I would probably distribute the negation so the whole thing reads:

```js
if (
  !isUserAuthenticated ||
  isEmpty(projectKeys) ||
  isNil(currentProjectKey)
) {
```